app/controllers/webhooks_controller.rb

```diff
@@ -69,7 +72,8 @@ def email_params
   end
 
   def authorized?
-    params[:auth].present? && params[:auth].secure_compare(ENV['WEBHOOK_AUTH_KEY'])
+    params[:auth].present? && params[:auth].
+      secure_compare(ENV['WEBHOOK_AUTH_KEY'])
```

Comment:
Might it make more sense to break this line (and others like it) at `&&`, rather than mid-method-call?